hypothesis-python/tests/cover/test_simple_characters.py

```diff
@@ -54,46 +53,47 @@ def test_when_nothing_could_be_produced():
 def test_characters_of_specific_groups():
     st = characters(whitelist_categories=('Lu', 'Nd'))
 
-    find(st, lambda c: unicodedata.category(c) == 'Lu')
-    find(st, lambda c: unicodedata.category(c) == 'Nd')
+    minimal(st, lambda c: unicodedata.category(c) == 'Lu')
```

Comment:
This is an existence claim which doesn't have any assertions about the return value, so perhaps it should use find_any instead of minimal?

Reply:
Good catch. Fixed and pushed.